core/jazz_es-kinesis-log-streamer/components/utils.js

```diff
@@ -0,0 +1,298 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+/**
+    Helper functions for Cloud Logs Streamer
+  @module: utils.js
+  @description: Helper functions for using Regex patterns, etc.
+  @version: 1.0
+**/
+
+const crypto = require('crypto');
+const logger = require("../components/logger.js");
+const config = require("../config/global_config.json")
+var truncate = require('unicode-byte-truncate');
+
+// Helper functions
+
+function getInfo(messages, patternStr) {
```

Comment:
indentation in this file is double compared to others

Reply:
Indentation fix is done.